test/Test/Network/Datadog/User.hs

```diff
@@ -0,0 +1,30 @@
+module Test.Network.Datadog.User (spec) where
+
+import Control.Concurrent (threadDelay)
+import Control.Lens ((^.))
+import Control.Monad (when)
+import Test.Hspec (Spec, describe, expectationFailure, it)
+
+import Network.Datadog (Environment, createEnvironment, loadKeysFromEnv)
+import Network.Datadog.User
+  ( UserHandle
+  , newUserHandle
+  , newUser
+  , updateUser
+  , disableUser
+  , loadUser
+  , loadUsers
+  )
+
+spec :: Spec
+spec = describe "User operations" $ do
+  it "Does user CRUD operations" $ do
+    let environment :: IO Environment
+        environment = createEnvironment =<< loadKeysFromEnv
+    env <- environment
+    let userEmail = "test@example.com"
+    user1 <- newUser userEmail
+    -- load an existing user
+    user2 <- loadUser env (user1 ^. newUserHandle)
```

Comment:
@iand675 I don't think I am using the right approach / not sure how to proceed with that test. Mind having a look when you have a chance? Thanks in advance.